src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -190,19 +196,18 @@ import scala.runtime.Statics
   *  }
   *  }}}
   *
-  *  The head, the tail and whether the list is empty or not can be initially unknown.
+  *  The head, the tail and whether the list is empty is initially unknown.
   *  Once any of those are evaluated, they are all known, though if the tail is
-  *  built with `#::` or `#:::`, it's content still isn't evaluated. Instead, evaluating
-  *  the tails content is deferred until the tails empty status, head or tail is
+  *  built with `#::` or `#:::`, its content still isn't evaluated. Instead, evaluating
+  *  the tail's content is deferred until the tail's empty status, head or tail is
```

Comment:
I think the oxford comma would be slightly clearer here

```suggestion
  *  the tail's content is deferred until the tail's empty status, head, or tail is
```